mobile/lib/ui/viewer/search_tab/people_section.dart

```diff
@@ -50,10 +52,12 @@ class _PeopleSectionState extends State<PeopleSection> {
     for (Stream<Event> stream in streamsToListenTo) {
       streamSubscriptions.add(
         stream.listen((event) async {
-          _examples = await widget.sectionType.getData(
-            context,
-            limit: kSearchSectionLimit,
-          );
+          _examples = await widget.sectionType
+              .getData(
+                context,
+                limit: kSearchSectionLimit,
+              )
+              .then((value) => List.from(value));
```

Comment:
Let's cast this to List<GenericSearchResult> instead of creating a new list?

Reply:
Done!